SCClassLibrary/Common/Quarks/QuarksGui.sc

```diff
@@ -149,9 +176,41 @@ QuarksGui {
 		detailView.update();
 	}
 
-	setMsg { |msg, color|
-		lblMsg.background = palette.button.blend(Color.perform(color ? 'yellow'), 0.2);
-		lblMsg.string = msg;
+	setMsg { |msg, style|
+		lblMsg.background = colors[style ? \default].bg;
+		lblMsg.stringColor = colors[style ? \default].text;
+		lblMsg.string = msg ? "";
+	}
+	checkForUpdates { |onComplete, onCancel|
+		this.runCancellable({
+			model.fetchDirectory(true);
+			model.checkForUpdates();
+		}, onComplete, onCancel)
+	}
+	installFolder {
+		FileDialog({ |path|
+				Quarks.install(path);
+				this.update;
+			},
+			nil, // cancel
+			2,  // QFileDialog::Directory
+			0,  // QFileDialog::AcceptOpen
+			true); // just first value in paths array
+	}
+	runCancellable { |fn, onComplete, onCancel|
```

Comment:
This let's the user command-. while something is running. But if the app thread is blocked due to getting output of a unixCmd then it does not let command-. stop it.